src/libcrypto/aes_api.c

```diff
@@ -1,9 +1,125 @@
 #include "include_asm.h"
+#include "ultratypes.h"
+#include "libcrypto/aes.h"
 
-INCLUDE_ASM("asm/non_matchings/libcrypto/aes_api", aesMakeKey);
+int aesMakeKey(AesKeyInstance* key, u8 direction, int keyLen, u8* keyMaterial) {
+    if (key == NULL) {
+        return -3;
+    }
+    if (direction >= 2U) {
```

Comment:
Do we need the U here on the 2? or is that convention here and I just am unaware